mobius/graphs.py

```diff
@@ -40,6 +42,87 @@ def clear_buffer(num, path_buffer):
         state = "horizontal"
         return num, path_buffer, state
 
+    def convert_list_to_dict(lst):
+        """Converts a list (with lists in) to a dictionary"""
+        new_dict = {}
+        num = 1
+        for item in lst:
+            new_dict[num] = item
+            num += 1
+        return new_dict
+
+    def check_output_order(output, save, output_folder):
+        """Checks the SVGs are in the correct order, and if not, reorders them.
+        Doesn't check first six figures order currently. Assumes these are correct.
+        """
+        output_order = []
+        output_order_list = [1, 2, 3, 4, 5, 6]
+
+        if len(output) > 6:
+
+            for block_start_num in range(1, 7, 6):
+                """Adds first 6 graphs in order as is"""
+                keys = list(range(block_start_num, block_start_num + 6))
+                block = [output.get(key) for key in keys]
+                for num in range(len(block)):
+                    output_order.append(block[output_order_list[num]-1])
+
+            for block_start_num in range(7, len(output), 12):
+                if len(output) - block_start_num > 6:
+                    """For pages with 12 graphs on"""
+                    keys = list(range(block_start_num, block_start_num + 12))
+                else:
+                    """For last page if it only has 6 graphs on"""
+                    keys = list(range(block_start_num, block_start_num + 6))
+
+                block = [output.get(key) for key in keys]
+
+                graph_count = 0
+                order_values = []
+                original_order = list(range(0, 12))
+
+                for num in range(len(block)):
+                    """Rough way of getting block graphs in order. Scale y value larger as it is the most important"""
+                    graph_count += 1
+                    order_value = (round(float(str(block[num][2][0].start).split('+')[-1].split('j')[0]), -2) * 1000) + \
+                                  float((str(block[num][2][0].start).split('+')[0].split('(')[-1]))
```

Comment:
Just a quick suggestion:

```suggestion
                    order_path = block[num][2][0]
                    order_value = round(order_path.start.imag, -2) * 1000 + order_path.start.real                   
```